Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/TaskCreateFragment.java

```diff
@@ -196,24 +233,13 @@ private void showStepsList(long taskId) {
         transaction.commit();
     }
 
+
+   @Override
     private void showMainMenu() {
         Intent intent = new Intent(getActivity(), MainActivity.class);
         startActivity(intent);
     }
 
-    @Override
-    public void onActivityResult(int requestCode, int resultCode, Intent data) {
-        super.onActivityResult(requestCode, resultCode, data);
-
-        if (filePickerProxy.isFilePicked(resultCode)) {
-            if (filePickerProxy.isPickFileRequested(requestCode, AssetType.PICTURE)) {
-                handleAssetSelecting(data, AssetType.PICTURE);
-            } else if (filePickerProxy.isPickFileRequested(requestCode, AssetType.SOUND)) {
-                handleAssetSelecting(data, AssetType.SOUND);
-            }
-        }
-    }
-
```

Comment:
handleAssetSelecting is not used now, but it was responsible for storing asset path in db.